test/unit/test-url-replacements.js

```diff
@@ -629,20 +628,6 @@ describes.sandboxed('UrlReplacements', {}, () => {
     });
   });
 
-  it('should replace CLIENT_ID with empty string for inabox', () => {
```

Comment:
We do have an integration test for this right?

Reply:
yes, i checked its here https://github.com/ampproject/amphtml/blob/a844c8fc66a2eb85fa610a0793293f8c73173392/test/integration/test-amp-inabox.js#L91